CHANGELOG.md

```diff
@@ -5,8 +5,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [2.0.0] - October 2, 2020
```

Comment:
So when does it get released?  I've always found this confusing when we've merged things but not released.

Reply:
Heh, I know it says tomorrow, but why tomorrow is my question?